tools/functions_missing_types.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
```

Comment:
It would be great if you could add a small docstring below this shebang explaining the purpose.